### ISSUE TITLE
CSRF token mismatch randomly on chrome but not insomnia

### ISSUE THREAD

**Issue by Xationch** [laravel/sanctum#135]
- Sanctum Version: v2.2.1
- Laravel Version: 7.9.2
- PHP Version: 7.4.3
- Database Driver & Version: Mysql
- SPA vue.js (installed with CLI not in the Laravel folder) 

### Description:
I used sanctum on homestead without problem until today or I suddenly got an error "CSRF token mismatch.". I am trying with Insomnia and I have no errors. I look at my routes and it's weird because the route is not in auth: sanctum middleware because there is no need to be authenticated to gain access. I change in my .env file the SANCTUM_STATEFUL_DOMAINS var by adding http:// and it's working...

To summarize. Whatever happens it works with Insomnia. On chrome I sometimes have to add the http:// to SANCTUM_STATEFUL_DOMAINS, sometimes no.

### Steps To Reproduce:

SANCTUM_STATEFUL_DOMAINS=app.lorem.test
SANCTUM_STATEFUL_DOMAINS=http://app.lorem.test

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by Xationch** [comment]
Apparently using SANCTUM_STATEFUL_DOMAINS=.lorem.test work also.
